label_studio/core/settings/base.py

```diff
@@ -598,6 +598,7 @@
 MEMBER_PERM = 'core.api_permissions.MemberHasOwnerPermission'
 RECALCULATE_ALL_STATS = None
 GET_STORAGE_LIST = 'io_storages.functions.get_storage_list'
+STORAGE_LOAD_TASKS_JSON = 'io_storages.utils._load_tasks_json'
```

Comment:
Usually the meaning of a function or method that starts with a `_` is "this should not be used outside of this class/module/whatever" - seems to me that we are treating this function as more of a public one, since it's being referred to in this other file. Would it be a pain to rename?